gdcdictionary/schemas/treatment.yaml

```diff
@@ -177,6 +179,126 @@ properties:
       - A081801
       - E4512
       - EA5142
+      - "901"
```

Comment:
do the new protocol_identifier enums need to be sorted alphabetically together with the existing ones?

Reply:
yes the script should do that... but it is having growing pains and not doing that... thanks for pointing out! alphabetized/numerically ordered 👍